src/DssSpell.t.sol

```diff
@@ -897,4 +946,49 @@ contract DssSpellTest is DssSpellTestBase {
         assertEq(art, 0, "RWA014/bad-art-after-spell");
         assertEq(ink, lockAmount, "RWA014/bad-ink-after-spell"); // Whole unit of collateral is locked
     }
+
+    function testSparkLendCollateralOnboarding() public {
+        // Configuration masking parameters pulled from https://github.com/aave/aave-v3-core/blob/62dfda56bd884db2c291560c03abae9727a7635e/contracts/protocol/libraries/configuration/ReserveConfiguration.sol
+        uint256 BORROWABLE_IN_ISOLATION_MASK = 0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFDFFFFFFFFFFFFFFF;
+        PoolLike pool = PoolLike(0x26ca51Af4506DE7a6f0785D20CD776081a05fF6d);
+        address token = addr.addr("GNO");
+        MedianAbstract medianizer = MedianAbstract(0x0cd01b018C355a60B2Cc68A1e3d53853f05A7280);
+        address oracleAdapter = 0xa2B52104c454D3f6717028783695de985C1CfFdb;
+        address interestRateStrategy = 0xE7Fe5041ec55c229fb41fD9183E5bc24B5E34959;
+        address wstETH = 0x6E4F1e8d4c5E5E6e2781FD814EE0744cc16Eb352;
+
+        PoolLike.ReserveData memory daiReserveData = pool.getReserveData(address(dai));
+        PoolLike.ReserveData memory tokenReserveData = pool.getReserveData(token);
+        assertEq((daiReserveData.configuration & ~BORROWABLE_IN_ISOLATION_MASK) != 0, false);
+        assertTrue(tokenReserveData.aTokenAddress == address(0));   // Not set yet
```

Comment:
The comment is not 100% clear. Should it be addressed later? should it be not addressed on goerli? ..etc.